business/health.go

```diff
@@ -287,37 +289,40 @@ func fillWorkloadRequestRates(allHealth models.NamespaceWorkloadHealth, rates mo
 	}
 }
 
-func (in *HealthService) getServiceRequestsHealth(namespace, service, rateInterval string, queryTime time.Time) (models.RequestHealth, error) {
+func (in *HealthService) getServiceRequestsHealth(namespace, service, rateInterval string, queryTime time.Time) (models.RequestHealth, []models.Threshold, error) {
 	rqHealth := models.NewEmptyRequestHealth()
+	srv, _ := in.businessLayer.Svc.GetServiceDefinition(namespace, service)
 	inbound, err := in.prom.GetServiceRequestRates(namespace, service, rateInterval, queryTime)
 	for _, sample := range inbound {
 		rqHealth.AggregateInbound(sample)
 	}
-	return rqHealth, err
+	return rqHealth, generateThreshold(inbound, namespace, service, "service", srv.Service.Labels), err
 }
 
-func (in *HealthService) getAppRequestsHealth(namespace, app, rateInterval string, queryTime time.Time) (models.RequestHealth, error) {
+func (in *HealthService) getAppRequestsHealth(namespace, app, rateInterval string, queryTime time.Time) (models.RequestHealth, []models.Threshold, error) {
 	rqHealth := models.NewEmptyRequestHealth()
+	appInfo, _ := fetchNamespaceApps(in.businessLayer, namespace, app)
 	inbound, outbound, err := in.prom.GetAppRequestRates(namespace, app, rateInterval, queryTime)
 	for _, sample := range inbound {
 		rqHealth.AggregateInbound(sample)
 	}
 	for _, sample := range outbound {
 		rqHealth.AggregateOutbound(sample)
 	}
-	return rqHealth, err
+	return rqHealth, generateThreshold(inbound, namespace, app, "app", getLabelsApp(appInfo[app].Workloads, appInfo[app].Services)), err
```

Comment:
Thresholds are checked only for inbound here ... I think it makes sense to define some thresholds only for inbounds, but then we'd also like to define some thresholds for outbounds. In other words, we could have something similar to your "kind", but for traffic direction: either "inbound", "outbound" or "both".